src/test/java/uk/gov/companieshouse/model/validator/EndDateValidatorTest.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.companieshouse.model.validator;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import java.time.LocalDateTime;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import uk.gov.companieshouse.TestData;
+import uk.gov.companieshouse.TestUtil;
+import uk.gov.companieshouse.model.Appeal;
+
+public class EndDateValidatorTest
+{
+
+    private EndDateValidator endDateValidator;
+    private Appeal appeal = new Appeal();
+
+    @BeforeEach
+    private void setup() {
+        endDateValidator = new EndDateValidator();
+        appeal = TestUtil.createAppeal(TestUtil.buildCreatedBy(),
+            TestUtil.createReasonWithIllness());
+        appeal.setCreatedAt(LocalDateTime.now());
+    }
+
+    @Test
+    void shouldReturnStringIfEndDateIsEmpty(){
+        appeal.getReason().getIllness().setIllnessEnd(null);
+        appeal.getReason().getIllness().setContinuedIllness(false);
+        assertEquals("Unable to validate. End date is empty",
```

Comment:
Instead of having a string here, in the class being tested, add the error string as a **public static final** and reference that. Same for all the error messages

Reply:
Makes sense, will do that now